augur/tree.py

```diff
@@ -32,7 +32,9 @@
     # with -fast, although there's probably no particular reason we have to.
     # Refer to the handling of -fast in utils/tools.cpp:
     #   https://github.com/Cibiv/IQ-TREE/blob/44753aba/utils/tools.cpp#L2926-L2936
-    "iqtree": "-ninit 2 -n 2 -me 0.05",
+    # Increasing threads (nt) can cause IQtree to run longer, hence use AUTO by default
+    # Auto makes IQtree chose the optimal number of threads
+    "iqtree": "-ninit 2 -n 2 -me 0.05 -nt AUTO",
```

Comment:
I added `-redo` (see [#1037](https://github.com/nextstrain/augur/issues/1037))

Every invocation of `augur tree` is supposed to start fresh - we don't make use of IQtree checkpoints. So `-redo` is an obvious flag to set.

If you don't set this flag and abort a run halfway, IQtree can error because there are checkpoint files lying around.